charts/README.md

```diff
@@ -19,3 +19,52 @@ $ helm install azuredisk-csi-driver azuredisk-csi-driver-latest.tgz --namespace
+| `image.clusterDriverRegistrar.tag`                | cluster-driver-registrar docker image tag                  | v1.0.1                                                       |
+| `image.clusterDriverRegistrar.pullPolicy`         | cluster-driver-registrar image pull policy                 | IfNotPresent                                                 |
+| `image.csiResizer.repository`                     | csi-resizer docker image                                   | mcr.microsoft.com/oss/kubernetes-csi/csi-resizer             |
+| `image.csiResizer.tag`                            | csi-resizer docker image tag                               | v0.3.0                                                       |
+| `image.csiResizer.pullPolicy`                     | csi-resizer image pull policy                              | IfNotPresent                                                 |
+| `image.livenessProbe.repository`                  | liveness-probe docker image                                | mcr.microsoft.com/oss/kubernetes-csi/livenessprobe           |
+| `image.livenessProbe.tag`                         | liveness-probe docker image tag                            | v1.1.0                                                       |
+| `image.livenessProbe.pullPolicy`                  | liveness-probe image pull policy                           | IfNotPresent                                                 |
+| `image.nodeDriverRegistrar.repository`            | csi-node-driver-registrar docker image                     | mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar |
+| `image.nodeDriverRegistrar.tag`                   | csi-node-driver-registrar docker image tag                 | v1.2.0                                                       |
+| `image.nodeDriverRegistrar.pullPolicy`            | csi-node-driver-registrar image pull policy                | IfNotPresent                                                 |
+| `serviceAccount.create`                           | whether create service account of csi-azuredisk-controller | true                                                         |
+| `rbac.create`                                     | whether create rbac of csi-azuredisk-controller            | true                                                         |
+| `controller.replicas`                             | the replicas of csi-azuredisk-controller                   | 2                                                            |
+| `node.metricsPort`                                | the metrics port of node                                   | 29605                                                        |
+| `snapshot.create`                                 | whether enable snapshot feature                            | true                                                         |
```

Comment:
should be false by default, otherwise azure disk driver would depends on 1.17

Reply:
`snapshot.enabled`

Reply:
done